src/main/java/com/google/alpollo/SongDataBase.java

```diff
@@ -16,4 +17,14 @@ public static void saveSongRequest(Song song) {
   public static List<Song> topSongs() {
     return OfyService.ofy().load().type(Song.class).limit(TOP_SIZE).list();
   }
+
+  /** Save whole song info to database. */
+  public static void saveSongInfo(SongInfo songInfo) {
+    OfyService.ofy().save().entity(songInfo).now();
+  }
+
+  /** Returns the song info by song id. */
+  public static SongInfo getSongInfo(int id) {
```

Comment:
According to the documentation, id should have `long` type instead of `int`

Reply:
I'll let Sasha handle this one.

Reply:
I've done